### ISSUE TITLE
What exactly the system is ?

### ISSUE THREAD

**Issue by abdulsami34** [glenndehaan/csgo-matchmaking#2]
Can you please explain what the system is about? 
Is it like Faceit or similar? Does it have any web template as well?
If yes, Can you please post screenshots?

Regards,
Sami.

**Comment by glenndehaan** [comment]
My original plan was that this project could be used as an replacement for Faceit or ESEA. The idea was that people could run an own matchmaking software for their own servers. But after 2 weeks of development I found out that such a system is a bit complex en would require an almost rebuild of Steam to make this an success. So after my first prototype with contains of an console based app. Both client and server. With simple functionality. I put the project here so others can work further on it if they want. So no I don't have screenshots and it doesn't have a GUI or web interface.

**Comment by bboychris168** [comment]
Hey @glenndehaan I just saw this system and i was thinking i could give you a hand on some ideas if you are still interested. I have a project that could definitely work and its a much more simpler system than your current repo i dont mind sharing you my code. my discord is MayNgu#8666 if you want to talk to me there.

**Comment by glenndehaan** [comment]
Hey @bboychris168 i'm always up for a chat. I have to say however that i'm not playing any CS at the moment and i'm a bit packed with other projects. But a chat is always fine with me. I will add you shortly.

**Comment by SamuelGogarty** [comment]
> that such a system is a bit complex en would require an almost rebuild of Steam

I don't think so necessarily, instead of trying to figure out how to put together a cohesive friends list system, or going through the trouble of somehow integrating user's friends list through the steam API, a much more simple method would be to simply give people invite codes in which they can share with their friends directly (through steam DM's, discord, whatever). I think this would be sufficient enough to fill that gap.

regardless, sorry for necro'ing this issue.

**Comment by bboychris168** [comment]
@SamuelGogarty Theres a guy thats already made a really good discord bot that does everything from scrimming to a matchmaking style queue all in discord. 

https://github.com/thboss/g5-discord-bot
